utils.go

```diff
@@ -207,21 +207,13 @@ func parseMsg(msg *pb.WebcastResponse_Message, warnHandler func(...interface{}),
 			groups := u.HostGroup
 			for _, group := range groups {
 				for _, user := range group.Host {
-					urls := make([]string, 5)
-					for _, img := range user.Images {
-						urls = append(urls, img.UrlList...)
-					}
+					//TODO change to host type
 					users = append(users, &User{
 						ID:       int64(user.Id),
 						Username: user.ProfileId,
 						Nickname: user.Name,
-						ProfilePicture: &ProfilePicture{
```

Comment:
This is looking good, but we need to keep a picture here, if you just throw in the image urls into AvatarThumb that would be good enough for now.